.github/workflows/bazel.yml

```diff
@@ -119,8 +123,38 @@ jobs:
         uses: browser-actions/setup-edge@latest
         with:
           edge-version: ${{ inputs.browser-version || 'stable' }}
+      - name: Setup Python Release
+        if: inputs.language-release-setup == 'python'
+        run: |
+          latest_tag=$(git describe --tags --abbrev=0 --exclude="nightly")
+          ./scripts/github-actions/update-version.sh "$latest_tag"
+      - name: Install Python
+        if: inputs.language-release-setup == 'python'
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install Twine and Update .pypirc
+        if: inputs.language-release-setup == 'python'
+        run: |
+          pip install --upgrade pip
+          pip install --upgrade twine
+          echo "[distutils]" > ~/.pypirc
+          echo "index-servers =" >> ~/.pypirc
+          echo "    testpypi" >> ~/.pypirc
+          echo "[testpypi]" >> ~/.pypirc
+          echo "username:$PYPI_TOKEN" >> ~/.pypirc
+          echo "password:$PYPI_PASS" >> ~/.pypirc
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+          PYPI_PASS: ${{ secrets.PYPI_PASS }}
       - name: Run Bazel
         run: ${{ inputs.run }}
+      - name: Upload Python Artifacts
```

Comment:
Whoops, this needs to be changed to use the twine upload command. a sample upload command would be great